docs/pages/access-controls/teleport-policy/getting-started-policy.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Teleport Policy
+description: A reference for Access Graph with Teleport Policy.
+---
+
+Teleport Policy streamlines and centralizes access management across your entire infrastructure.
+
+Teleport Policy with Access Graph provides a visual representation of the relationships between
+users, roles, and resources in your organization. It can help you answer questions like:
+
+- What resources can a specific user access?
+- What users can access a specific resource?
+- What are the relationships between users, roles, and resources?
+
+## Getting started with Teleport Policy
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product that is only 
```

Comment:
I would remove the link here, since the audience for a marketing landing page would be different from the audience of the docs. Ideally, we'd provide the same information in this section of the docs as that section of the corporate site.